scripts/calendars.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
you updated lots for cldr locale data scripts, but tcs are only for localeinfo.
I think It needs to make sure all updated scripts are not broken in current related test cases.

Reply:
I thought some scripts aren't related localeinfo.json file generation. I realized I misunderstood. all scrips files for generating localeinfo.json.